src/interdiction/Interdiction.jl

```diff
@@ -0,0 +1,249 @@
+__precompile__(true)
+module Interdiction
+
+using LightGraphs
+using JuMP
+using MathProgBase
+
+import MathProgBase.SolverInterface.AbstractMathProgSolver,
+       JuMP.UnsetSolver
+
+export interdiction_flow, ⪷,
+       NetworkInterdictionProblem, AdaptiveFlowArcProblem, AdaptiveFlowPathProblem,
+       MultilinkAttackAlgorithm, BilevelMixedIntegerLinearProgram
```

Comment:
Sorry, I am traveling and my internet access is scarce. I lost it before seeing the results of Travis.

That being said, I think the problem is not an export problem. It is actually related to `LightGraphs.jl`. I suppose Travis use the last tagged version of each package being used. The function multiroute is only available on master branch in `LightGraphs.jl`.
If I add a `Pkg.checkout("LightGraphs")` in the test file, would it work?
Or a tagged version of `LightGraphs.jl` will do the trick?